### PR TITLE
[Notifier] Added redlink notifier

### PR DESCRIPTION
**Pull request by plotkabytes**
| Q             | A
| ------------- | ---
| Branch?       | 6.3
| Bug fix?      | no
| New feature?  | yes
| Deprecations? | no
| Tickets       | 
| License       | MIT
| Doc PR        | https://github.com/symfony/symfony-docs/pull/18263

Added [Redlink](https://docs.redlink.pl) notifier bridge

### CODE REVIEW

File: src/Symfony/Component/Notifier/Bridge/Redlink/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+DSN example
+-----------
+
+```
+REDLINK_DSN=redlink://API_TOKEN:APP_TOKEN@default?from=SENDER_NAME&version=VERSION
+```
+
+where:
+
+- `API_TOKEN` is your user API token, you can get it from user dashboard
```
Comment (OskarStark) [inline src/Symfony/Component/Notifier/Bridge/Redlink/README.md]:
```suggestion
- `API_TOKEN` is your user API token, you can get it from the user dashboard
```

File: src/Symfony/Component/Notifier/Bridge/Redlink/RedlinkTransport.php [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+/**
+ * @author Mateusz Żyła <https://github.com/plotkabytes>
+ */
+final class RedlinkTransport extends AbstractTransport
+{
+    protected const HOST = 'api.redlink.pl';
+
+    public function __construct(
+        #[\SensitiveParameter] private readonly string $apiToken,
```
Comment (OskarStark) [inline src/Symfony/Component/Notifier/Bridge/Redlink/RedlinkTransport.php]:
```suggestion
        #[\SensitiveParameter] 
private readonly string $apiToken,
```

File: src/Symfony/Component/Notifier/Bridge/Redlink/RedlinkTransport.php [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+        #[\SensitiveParameter] private readonly string $appToken,
```
Comment (OskarStark) [inline src/Symfony/Component/Notifier/Bridge/Redlink/RedlinkTransport.php]:
```suggestion
        #[\SensitiveParameter] 
private readonly string $appToken,
```

File: src/Symfony/Component/Notifier/Bridge/Redlink/RedlinkTransport.php [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+    public function supports(MessageInterface $message): bool
+    {
+        return $message instanceof SmsMessage;
+    }
+
+    public function __toString(): string
+    {
+        return sprintf(
+            'redlink://%s:%s@%s?from=%s&version=%s',
+            $this->apiToken,
```
Comment (OskarStark) [inline src/Symfony/Component/Notifier/Bridge/Redlink/RedlinkTransport.php]: Should not be exposed

File: src/Symfony/Component/Notifier/Bridge/Redlink/RedlinkTransport.php [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+            $this->appToken,
```
Comment (OskarStark) [inline src/Symfony/Component/Notifier/Bridge/Redlink/RedlinkTransport.php]: Should not be exposed

File: src/Symfony/Component/Notifier/Bridge/Redlink/composer.json [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=8.1",
+        "symfony/http-client": "^5.4|^6.0",
+        "symfony/notifier": "^6.2.7"
```
Comment (OskarStark) [inline src/Symfony/Component/Notifier/Bridge/Redlink/composer.json]:
```suggestion
        "symfony/notifier": "^6.3"
```

**Comment by OskarStark** [comment]
Can you please open a PR on the Symfony docs and the symfony/recipes repository? Thanks

**Comment by OskarStark** [comment]
Why did you close the PR?

**Comment by plotkabytes** [comment]
> Why did you close the PR?

Sorry, this was by mistake. I made suggested changes, and I will open PR on the Symfony docs and the symfony/recipes repository soon. Thanks!

**Comment by plotkabytes** [comment]
Done.
recipes: [symfony/recipes#1193](https://github.com/symfony/recipes/pull/1193)
docs: [symfony/symfony-docs#18263](https://github.com/symfony/symfony-docs/pull/18263)

> Can you please open a PR on the Symfony docs and the symfony/recipes repository? Thanks

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @plotkabytes.

**Pull request merged**